.github/workflows/verify-pr-commit.yml

```diff
@@ -429,7 +429,7 @@ jobs:
         run: |
           chmod 755 ${{env.BIN_FILENAME}}
       - name: Output Metadata
-        run: ${{env.BIN_DIR}}/${{env.BIN_FILENAME}} export-metadata  --chain=frequency-local --tmp ./js/api-augment/metadata.json
+        run: ${{env.BIN_DIR}}/${{env.BIN_FILENAME}} export-metadata  --chain=dev --tmp ./js/api-augment/metadata.json
```

Comment:
I would expect this to be `frequency-rococo-testnet` or `frequency-rococo-local`

Reply:
I'd like to better understand the reasoning behind your expectation here.  It seems to me that it should be either `dev` or `frequency-no-relay` (they are aliases.)

Reply:
Fixed